### PR TITLE
Feature/merge pame database

### PR DESCRIPTION
**Pull request by drewpotter**
This is the PR of cherry-pick commits from the branch I am working on locally, due to the need to use Docker locally and due to the fact that the Docker branch is not ready for merge. I am cherry-picking the work to merge the PAME database over to Protected Planet.

### CODE REVIEW

**Comment by Levia** [comment]
@drewpotter Any reason why the "do not merge" label is still in here?

**Comment by drewpotter** [comment]
Sorry @Levia I put that in place when I had those random test failures with PP-API now that those are not consistent I feel like I can remove the "do not merge" label now.

File: .gitmodules [hunk 1]
```diff
@@ -1,4 +1,4 @@
 [submodule "db"]
 	path = db
   url = https://github.com/unepwcmc/protectedplanet-db.git
-  branch = master
+  branch = merge-pame-database
```
Comment (Levia) [inline .gitmodules]: Can you make sure to change this to master again once everything is merge into master in the pp-db repo please?
Happy to merge after that.

Comment (drewpotter) [reply .gitmodules]: Sure thing! I will change it back to `master` once pp-db is merged into master!

**Pull request merged**